dist/rpm/os-autoinst.spec

```diff
@@ -87,6 +87,11 @@ Source0:        %{name}-%{version}.tar.xz
 %else
 %define python_support_requires %{nil}
 %endif
+# The following line is generated from dependencies.yaml
+%define lua_support_requires perl(Inline::Lua)
+%else
+%define lua_support_requires %{nil}
+%endif
```

Comment:
This seems to miss an `$if %{with lua_support}` and of course also a preceding `%bcond_…` macro. I think this should be done like we already conditionally enable Python support.